packages/forms-web-app/__tests__/unit/routes/index.test.js

```diff
@@ -40,6 +40,6 @@ describe('routes/index', () => {
     expect(use).toHaveBeenCalledWith('/before-you-start', fullAppealRouter);
     expect(use).toHaveBeenCalledWith('/document', documentRouter);
 
-    expect(use.mock.calls.length).toBe(9);
+    expect(use.mock.calls.length).toBe(11);
```

Comment:
is this the only change that's supposed to be there?

Reply:
Householder-planning route test added.